datahub-web-react/src/app/entity/user/UserInfoSideBar.tsx

```diff
@@ -0,0 +1,347 @@
+                            </Paragraph>
+                        </AboutSectionText>
+                    </AboutSection>
+                    <Divider className="divider-groupsSection" />
+                    <GroupsSection>
+                        Groups
+                        <TagsSection>
+                            {groupsDetails?.relationships.length === 0 && <NoDataFound>No Groups</NoDataFound>}
+                            {!groupSectionExpanded &&
+                                groupsDetails?.relationships.slice(0, 2).map((item) => {
+                                    return (
+                                        <Link to={entityRegistry.getEntityUrl(EntityType.CorpGroup, item.entity.urn)}>
+                                            <Tags>
+                                                <Tag>
+                                                    {item.entity.info.displayName || item.entity.name || <EmptyValue />}
+                                                </Tag>
+                                            </Tags>
+                                        </Link>
+                                    );
+                                })}
+                            {groupSectionExpanded &&
+                                groupsDetails?.relationships.length > 2 &&
+                                groupsDetails?.relationships.map((item) => {
+                                    return (
+                                        <Tags>
+                                            <Tag>
+                                                {item.entity.info.displayName || item.entity.name || <EmptyValue />}
+                                            </Tag>
+                                        </Tags>
+                                    );
+                                })}
+                            {!groupSectionExpanded && groupsDetails?.relationships.length > 2 && (
+                                <GroupsSeeMoreText onClick={() => setGroupSectionExpanded(!groupSectionExpanded)}>
+                                    {`+${groupsDetails?.relationships.length - 2} more`}
+                                </GroupsSeeMoreText>
+                            )}
+                        </TagsSection>
+                    </GroupsSection>
+                </SideBarSubSection>
+                <EditProfileButton>
+                    <Button icon={<EditOutlined />} onClick={() => showEditProfileModal(true)}>
+                        Edit Profile
+                    </Button>
+                </EditProfileButton>
+            </SideBar>
+            {/* Modal */}
+            <UserEditProfileModal
+                visible={editProfileModal}
+                onClose={() => showEditProfileModal(false)}
+                onSave={() => {
+                    refetch();
+                }}
+                editModalData={getEditModalData()}
```

Comment:
Why call a function here? Why not just have a const variable above

Reply:
Sure, I saw few logic's were written in same manner, so I kept it similar to them. 
The required changes you have asked are added.